inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -176,6 +176,8 @@ target_link_libraries(${TARGET_NAME}_s PRIVATE openvino::itt ${CMAKE_DL_LIBS} ${
 
 target_compile_definitions(${TARGET_NAME}_s PUBLIC USE_STATIC_IE)
 
+set_target_properties(${TARGET_NAME}_s PROPERTIES EXCLUDE_FROM_ALL ON)
```

Comment:
How will pre-processing work? As I see [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/preprocessing/CMakeLists.txt#L156) it is required `inference_engine_s`. Or CMake automatically build such dependencies?

Also should we exclude other static libraries (for example pre-processing library)?

Reply:
`EXCLUDE_FROM_ALL` means not to build with default (`all`) target. But if some other targets require such "optional" target, it still will be built. 

I just want to avoid cases when we build `MKLDNNPlugin_obj`, but nobody links it.